ast/parser.go

```diff
@@ -849,6 +849,21 @@ func (p *Parser) parseLiteral() (expr *Expr) {
 					p.hint("`import future.keywords.every` for `every x in xs { ... }` expressions")
 				}
 			}
+			// If we can't find `every or some` still check if import future.keywords.in
+			if term, ok := expr.Terms.(*Term); ok && (!Var("every").Equal(term.Value) && !Var("some").Equal(term.Value)) {
+				t := p.save()
+				p.restore(s)
+				p.scan()
+				if term := p.futureParser().parseTermInfixCall(); term != nil {
+					if call, ok := term.Value.(Call); ok {
+						switch call[0].String() {
+						case Member.Name, MemberWithKey.Name:
+							p.hint("`import future.keywords.in` for `x in xs { ... }` expressions")
```

Comment:
🔍 There's no `x in xs { ... }` expression in Rego. We have those:

````rego
some x in xs
x in xs
every x in xs { ... }
```
(all optionally taking a key argument, like `some k, v in obj`)

So I guess it would be better to drop the ` { ... }` here:
```suggestion
							p.hint("`import future.keywords.in` for `x in xs` expressions")
```